plugins/MusicPlayer/player.js

```diff
@@ -11,12 +11,12 @@ let MAXIMUM_SONG_BUFFER_SIZE = (options && options.maxSongBufferSize) || 1024 *
 let SONG_BUFFER_TIME = (options && options.songBufferTimeMS) || 1000;
 
 exports.commands = [
-    "play",
-    "skip",
+    "oynat",
+    "geç",
     "queue",
     "dequeue",
-    "pause",
-    "resume"
+    "durdur",
+    "devamet"
```

Comment:
Changing from english to another language is not supported in pull requests. Try making a lang file for your language to have support added.